File: Patterns/Cpp/Pattern7.cpp
/*
Print the following pattern:
Input: 
5

Output:
*
    *
        *
            *
                *

Note: This question is also available in Java.
*/

#include <iostream>
using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;
    for(int i = 0; i < n; i++) {
        for(int j = 0; j < n; j++) {
            if(i == j)
                cout << "*\t";
            else
                cout << "\t";
        }
        cout << endl;
    }
}

File: Patterns/Java/Pattern7.java
/*
Print the following pattern:
Input: 
5

Output:
*
    *
        *
            *
                *

Note: This question is also available in Cpp.
*/

import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int row = scn.nextInt();
        for(int i = 0; i < row; i++) {
            for(int j = 0; j < row; j++) {
                if(i == j)
                    System.out.print("*\t");
                else
                    System.out.print("\t");
            }
            System.out.println("");
        }
    }
}
